Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -136,6 +136,13 @@ function ScreenSpaceCameraController(scene) {
    * @default {@link Number.POSITIVE_INFINITY}
    */
   this.maximumZoomDistance = Number.POSITIVE_INFINITY;
+  /**
+   * A multiplier for the speed at which the camera will zoom.
+   * @type {Number}
+   * @default 5.0
+   */
+  this.zoomFactor = 5.0;
+  this._zoomFactor = this.zoomFactor;
```

Comment:
@brianpmaher how about using getter and setter for the public variable? The setter will also set the private one. Do not know if this is the desired way for backward compatibilty though 🤔

Reply:
I think either way should be fine.  I'm following the convention set by those before me here.  

Here's a few examples below where it was done this way: 
https://github.com/CesiumGS/cesium/pull/9145/files#diff-3e56d3751092b5a52b53781e4afee4f7L224
https://github.com/CesiumGS/cesium/pull/9145/files#diff-3e56d3751092b5a52b53781e4afee4f7L231
https://github.com/CesiumGS/cesium/pull/9145/files#diff-3e56d3751092b5a52b53781e4afee4f7L239

Reply:
@bampakoa I think I finally understood what you were saying about the getter and setter to maintain the private attribute any time the public one updated (and vice-versa).  I got to thinking that adds some unnecessary complexity when changing the field, so I decided to just remove the private attribute instead.  I'll update the CHANGES.md to reflect that.